compiler/fir/checkers/checkers-component-generator/src/org/jetbrains/kotlin/fir/checkers/generator/diagnostics/FirDiagnosticsList.kt

```diff
@@ -900,6 +900,13 @@ object DIAGNOSTICS_LIST : DiagnosticList("FirErrors") {
             parameter<Symbol>("symbol")
         }
     }
+
+    val JVM by object : DiagnosticGroup("jvm") {
+        val JAVA_TYPE_MISMATCH by error<KtExpression> {
+            parameter<ConeKotlinType>("actualType")
```

Comment:
This is super-nitty, but AFAICT for every other expected/actual in diagnostics, expected comes first before actual. I noticed this because in the new error message, `{1}` comes before `{0}`. 🙂

Reply:
Yeah, not sure why it was reverted in FE1.0. But I can swap them.

Reply:
I wish the quick reaction emoji has 🤦